frontend/src/components/Reports/auditTrailReport/AuditTrail.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import {
+  Form,
+  Heading,
+  Select,
+  SelectItem,
+  Grid,
+  Column,
+  Section,
+  Button,
+  Loading,
+} from "@carbon/react";
+import "../../Style.css";
+
+import { AlertDialog } from '../../common/CustomNotification';
+import config from "../../../config.json";
+import CustomLabNumberInput from '../../common/CustomLabNumberInput';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+
+const AuditTrail = () => {
+    const [labNo, setLabNo] = useState("");
+    const [isLabNoError, setIsLabNoError] = useState(false);
+    const [isLoading, setIsLoading] = useState(false);
+    const [showNotification, setShowNotification] = useState(false);
+
+    const intl = useIntl();
+
+    const handleViewReport = () => {
+        if(labNo.length === 0){
+            setIsLabNoError(true);
+            return;
+        }
+
+        setIsLoading(true);
+        const url = config.serverBaseUrl + `/AuditTrailReport?accessionNumberSearch=${labNo}`;
+
+        window.open(url);
+        setIsLoading(false);
+        setShowNotification(true);
+    }
+
+    return (
+        <>
+        <br />
+        <Grid fullWidth={true}>
```

Comment:
Well done. Do you mind sharing the final vedio please.

Reply:
Hi @sherrif10 
The vidoe is what I included in the description above